### ISSUE TITLE
508-defect-2 [COGNITION, SCREENREADER]: Facility Locator - Can we improve the alt text on Mapbox maps for JAWS, other screen readers

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#2754]
@1Copenut commented on [Mon Feb 18 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16987)

## Issue
When users focus the Mapbox map to zoom in or out, screen readers hear a long string of text that makes it hard to understand they are hearing the zoom labels read out. JAWS in particular reads out the following:

> "Zoom in zoom out Leaflet vertical bar Map data copyright OpenStreetMap contributors, \ CC-BY-SA, \ Imagely copyright Mapbox trademark".

* The Zoom In / Out controls should be distinctly separated for interaction.
* The map attribution should be separated from the map controls, possibly wrapped in a `<cite>` element.

## Related Issue:
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/514

## Acceptance Criteria

* As a screen reader user, I want to hear the interactive Zoom In/Out controls separately from the map attribution content.
* As a screen reader user, I want to hear the attribution content separately from the map interactive controls.

## Environment
* Windows 7 Enterprise Edition
* Internet Explorer 11
* JAWS 2018
* https://staging.va.gov/find-locations/?address=Dallas%2C%20TX&context=Dallas%2C%20Texas%2C%20United%20States&facilityType=all&location=32.7763%2C-96.7969&page=1&serviceType&zoomLevel=9

## Steps to Recreate
1. Enter IP address listed above in browser
2. Start screenreading device listed in Environment
3. Tab through the page (many links) until you are focused on the map on the right. Confirm hearing the string called out above.

## Possible fix

* The map attribution could possibly be wrapped in a `<cite>` element.

```html
<figure>
  <canvas <!-- map goes here -->></canvas>
  <figcaption>
      <cite>Attribution: <a href="#">Leaflet</a>,  Map data copyright <a href="#">OpenStreetMap</a> contributors, CC-BY-SA, Imagely, copyright <a href="#">Mapbox</a> trademark</cite>
  </figcaption>
</figure>
```

## WCAG or Vendor Guidance (optional)
* [Link Purpose (In Context): Understanding SC 2.4.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html)

---
![picture1](https://user-images.githubusercontent.com/934879/52979492-7ce76e00-339b-11e9-9f71-3de5d8703465.png)

---

@mmiddaugh commented on [Tue Oct 22 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16987#issuecomment-545136371)

@brangi , here is one to evaluate for level of effort and usability impact

**Comment by 1Copenut** [comment]
@mmiddaugh I've added a related ticket in the description, #514 .

**Comment by mmiddaugh** [comment]
Need to get more information on Mapbox obligations/talk with Sim
@davidconlon

**Comment by davidconlon** [comment]
@mmiddaugh Found this on Mapbox
https://docs.mapbox.com/help/how-mapbox-works/attribution/

Can we move this forward to determine a mutually acceptable solution?

**Comment by mmiddaugh** [comment]
<attr>Copyright <a href=“link”>Leaflet</a> | <a href=“#”>Other</a> | any other stuff</attr>

**Comment by mmiddaugh** [comment]
This has been resolved by the Mapbox upgrade
